egs/wsj/s5/steps/dur_model/make_nnet_config.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+learning_rate=0.001
+natural_gradient=false
```

Comment:
Is there a reason you default natural_gradient to false?  I have never seen a situation where it was not helpful.